cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -176,19 +176,6 @@ namespace
         return false;
     }
 
-    void writeParamAllocateCode(
-        Output& out,
-        const TypePtr& type,
-        bool optional,
-        const string& scope,
-        const string& fixedName,
-        const MetadataList& metadata,
-        TypeContext typeCtx)
-    {
-        string s = typeToString(type, optional, scope, metadata, typeCtx);
-        out << nl << s << ' ' << fixedName << ';';
-    }
```

Comment:
This function is only 2 lines... but takes 7 arguments. I inlined it in the few places where it was used.